BNO.cpp

```diff
@@ -3,16 +3,18 @@
  * the robot angle, write on screen, and
  * calibrate all sensors.
  * To get more information, go to BNO.h file.
+ * Marlon Romo (MarlonB500).
 */
 #include "BNO.h"
 
-BNO::BNO() {
+BNO::BNO(Multiplexor *multiplexor) {
+  I2C_ = multiplexor;
   bno_ = Adafruit_BNO055();
   Serial.println("Orientation Sensor Test");
   Serial.println("");
-  // i2c_->tcaselect(2);
+  I2C_->tcaselect(2);
```

Comment:
is it always going to be a 2? Make it a constant please, or even better, a parameter that receives the BNO constructor.